app/src/main/java/ch/epfl/sdp/appart/user/UserViewModel.java

```diff
@@ -29,14 +32,18 @@
 
     final DatabaseService db;
     final LoginService ls;
+    final LocalDatabaseService localdb;
 
     @Inject
-    public UserViewModel(DatabaseService database, LoginService loginService) {
+    public UserViewModel(DatabaseService database, LoginService loginService,
+                         LocalDatabaseService localdb) {
 
         this.db = database;
         this.ls = loginService;
+        this.localdb = localdb;
     }
 
+    // TODO is this needed?
```

Comment:
Is it ? ;)

Reply:
I don't know, in my opinion it isn't but I found it there and forgot to ask carlo about it ahah